lib/Elevate/Blockers/Repositories.pm

```diff
@@ -147,25 +147,19 @@ sub _blocker_invalid_yum_repos ($self) {
         }
 
         return 0 unless _yum_status_hr_contains_blocker($status_hr);
-
-        for my $unsupported_repo ( @{ $self->{_yum_repos_unsupported_with_packages} } ) {
```

Comment:
I'm unclear why we're altering this. This information is actually reporting well. Are you just moving the code? Right now I get a good short subject unique to each repo name. I don't want to aggregate them into one report.

Reply:
This was taken out because it would have added in a second blocker when this specific one is reported on `_check_yum_repos`.